Makefile

```diff
@@ -71,9 +86,11 @@ build/sphinx/latex/khmer.pdf: $(SOURCES) doc/conf.py $(wildcard doc/*.txt)
 cppcheck-result.xml: $(CPPSOURCES)
 	${CPPCHECK} --xml-version=2 2> cppcheck-result.xml
 
+## cppcheck             : run static analysis on C++ code
 cppcheck: $(CPPSOURCES)
 	${CPPCHECK}
 
+## pep8                 : run static analysis on Python code
```

Comment:
static analysis is a bit strong here; `pep8` just enforces the style guide. `pylint` comes closer to static analysis